web/styles/portico/portico.css

```diff
@@ -16,6 +16,12 @@ html {
     position: fixed;
 }
 
+:target {
+    /* Keep elements targeted by URL fragments visible below the
+    floating top menu; coordinate the value here with the menu's height. */
+    scroll-margin-top: 60px;
+}
```

Comment:
This selector is too broad. We apply `portico.css` on basically every page that isn’t the main app, including the login, billing, and help pages, which have a variety of different scrolling setups.

Reply:
Got it. So it seems like no only should the selector be more targeted, but maybe it belongs in the `landing_page.css` file, too?

Since the original for this referenced case-study pages, maybe `.case-study-page :target` is the selector to start with. But I'll ask [over on CZO](https://chat.zulip.org/#narrow/stream/9-issues/topic/landing.20page.20anchor.20links/near/1566217).

Reply:
I think there's two possible changes here. One idea would be to add this selector in a CSS file that's included precisely for the pages where this is needed; but I'm not sure how to compute that easily. 

I think `.portico-landing` might be the right class for "page with landing page navigation".

```
$ git grep  --files-with-match 'class=.*portico-landing'  > /tmp/a
$ git grep  --files-with-match "include 'zerver/landing_nav.html" >/tmp/b
$ diff -ur /tmp/a /tmp/b 
--- /tmp/a	2023-05-10 12:08:31.825913529 -0700
+++ /tmp/b	2023-05-10 12:07:43.100974445 -0700
@@ -1,6 +1,5 @@
 templates/corporate/apps.html
 templates/corporate/attribution.html
-templates/corporate/billing.html
 templates/corporate/case-studies/asciidoctor-case-study.html
 templates/corporate/case-studies/atolio-case-study.html
 templates/corporate/case-studies/end-point-case-study.html
@@ -12,7 +11,6 @@
 templates/corporate/case-studies/ucsd-case-study.html
 templates/corporate/communities.html
 templates/corporate/development-community.html
-templates/corporate/event_status.html
 templates/corporate/features.html
 templates/corporate/for/business.html
 templates/corporate/for/communities.html
@@ -29,7 +27,6 @@
 templates/corporate/self-hosting.html
 templates/corporate/team.html
 templates/corporate/try-zulip.html
-templates/corporate/upgrade.html
 templates/corporate/values.html
 templates/corporate/why-zulip.html
 templates/corporate/zephyr.html
```

The three pages in the diff are billing system pages that I would expect to not have the `portico-landing` class at all, I'll change that, but then after that I think this is the correct rule.

Reply:
https://github.com/zulip/zulip/pull/25518 fixes that issue; following it the above diff is the empty set.